Modules/Misc/srum-dump.mkape

```diff
@@ -1,14 +1,15 @@
 Description: 'SRUM-dump: Dump contents of the srum database'
 Category: SystemActivity
-Author: Brian Maloney
-Version: 1.0	
-Id: 8e0a2314-79e4-48f2-81ad-a0719e8af680
+Original Author: Brian Maloney
+Author: Jay Houlden
+Version: 1.1
+Id: c594e592-b740-4d01-a0f8-799ff9bac97a
 BinaryUrl: https://github.com/MarkBaggett/srum-dump/raw/python3/srum_dump_csv.exe
 ExportFormat: CSV
 Processors:
     -
         Executable: srum_dump_csv.exe
-        CommandLine: -i %sourceDirectory% -t SRUM_TEMPLATE.xlsx -r %sourceDirectory% -o %destinationDirectory% -q
+        CommandLine: -i %sourceDirectory%\Windows\System32\SRU\SRUDB.dat -t SRUM_TEMPLATE.xlsx -r %sourceDirectory%\WINDOWS\System32\config\SOFTWARE -o %destinationDirectory% -q
```

Comment:
I've never ran this before so I'm asking purely out of curiosity. Was this incorrect syntax previously? Is this command more verbose while accomplishing the same thing? Or is it adding functionality? Figured I'd ask since you have the Author thing to fix anyways haha

Reply:
Incorrect syntax. In the comments it says you have to set the variables to the correct files. So why don't we just set them that way in the first place.  I think it assumed you had all the files in the %SourceDirectory%.  But it doesn't look recursively like Amcacheparser or other modules.  I tried using %sourceFile% and FileMask too but didn't work.

Reply:
Also when i update a kape file do I generate a new GUID or is that only for new files?

Reply:
New GUID is only for new files. Just replacing content within a Target/Module doesn't require a new GUID.

Reply:
> 
> 
> Incorrect syntax. In the comments it says you have to set the variables to the correct files. So why don't we just set them that way in the first place. I think it assumed you had all the files in the %SourceDirectory%. But it doesn't look recursively like Amcacheparser or other modules. I tried using %sourceFile% and FileMask too but didn't work.

Thanks for looking into it and improving it.